include/raft.h

```diff
@@ -561,6 +569,11 @@ typedef struct
     void* udata_address;
 } raft_node_configuration_t;
 
+/** Set the global logging level.
+ * Messages at levels lower than this level will not be logged. Default to
```

Comment:
I think levels _higher_ than this level will not be logged

Suggested wording (to be accurate relative to the code logic, and to remove the words "not be" for simplicity)
"Only those messages at levels less than or equal to this level will be logged"

Reply:
Thanks. I'll eliminate the ambiguity here. (By "lower" I meant logical, instead of numerical, levels. But as you pointed out, one can also take it to mean numerical values or even frequencies.)